tests/apps/orderform/conftest.py

```diff
@@ -50,30 +53,38 @@ def fixture_minimal_excel_sample() -> dict:
     }
 
 
-@pytest.fixture(name="mip_order_parser")
+@pytest.fixture(scope="session", name="mip_order_parser")
 def fixture_mip_order_parser(mip_orderform: str) -> ExcelOrderformParser:
     """Return a orderform parser that have parsed a mip orderform in excel format"""
     order_form_parser = ExcelOrderformParser()
```

Comment:
Add type hint for this line and the other identical lines in the test module. Or make it an own fixture returning the  order_form_parser and import and use it directly. This is really optional 😄

Reply:
Do you mean like this?
`order_form_parser: ExcelOrderformParser = ExcelOrderformParser()`

Reply:
👍 . Let's M&D. Great work!

Reply:
Perfect. I need AZ for some customer emailing so the deploy will probably be tomorrow morning 👍